src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -46,12 +46,20 @@ export class OverflowMenuPane extends Dialog {
 			this.dialogConfig.menuLabel = this.i18n.get().OVERFLOW_MENU.OVERFLOW;
 		}
 
-		setTimeout(() => this.listItems()[0].focus());
+		setTimeout(() => {
+			let options = Array.from<any>(this.dialog.nativeElement.querySelectorAll("*"));
```

Comment:
This should probably be 

```suggestion
			let options = Array.from<any>(this.dialog.nativeElement.querySelectorAll(".bx--overflow-menu-options__btn"));
```

Since the options need that class to look like anything useful